frameworks/keyed/angular-cf-new-nozone/src/app/app.component.ts

```
@@ -0,0 +1,87 @@
+    }
+    runLots() {
+        this.data = this.buildData(10000);
+        this.selected = undefined;
+    }
+    clear() {
+        this.data = [];
+        this.selected = undefined;
+    }
+    swapRows() {
+        if (this.data.length > 998) {
+            var a = this.data[1];
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)